contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6159,11 +6162,40 @@ transformSelectIntoStmt(CreateTableAsStmt *stmt)
 			}
 			else
 			{
+				AlterTableStmt *newstmt;
```

Comment:
We should add else if () ?? some condition to enter this block here, we can't make all the else conditions into this block actually.